test/library/library.go

```diff
@@ -0,0 +1,35 @@
+package library
+
+import (
+	"crypto/rand"
+	"fmt"
+	"math"
+	"math/big"
+	"path"
+	"regexp"
+	"runtime"
+	"testing"
+	"time"
+)
+
+var (
+	waitPollInterval = time.Second
+	waitPollTimeout  = 10 * time.Minute
```

Comment:
we have `ForeverTestTimeout` in `k8s.io/apimachinery/pkg/util/wait`.

Reply:
`ForeverTestTimeout` is only 30 seconds.